### ISSUE TITLE
[Research] Admin needs

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/reserve-rec-admin#10]
#### Description:

[Description of the task]

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
This will be directly influenced by data modelling.
Majority of admin will be a mirror of what we offer on our api. - CRUD operations - create/update/delete
- will have this on all of our objects - eg: protected area layer, facility, etc, whatever we are calling them.
- In the past in the api side we ignored making crud operations (eg: A&R) - but now it bites us in the butt

Regardless, we need to make crud operations for it. Todo make ticket - for future work.

side note - right now - data model shows what's relevant now. Once confident, they will make api endpoints to facilitate creating/updating/deleting on those layers (sub area area). **So currently, solidifying data model** --- todo create tickets.

**Comment by Dianadec** [comment]
Refinement notes:
- Cam - idea to include in this ticket - look into data types and build front-ends to support their CRUD endpoints.
- todo: discuss timing and prioritization of this work (possibly seeing this as complementary of the user story session for admin side of things).
- ... and also expand to Park Operators as a next step to this first step.

**Comment by Dianadec** [comment]
@LindsayMacfarlane  to chat with NDR about her supporting this work

**Comment by Dianadec** [comment]
Sprint planning notes:
- Separate ticket specifically for Information Architecture and refine for this sprint. #13 
- this will help use the data we already have to work from. So, next step is to break it down into an IA.
